services/cart.js

```diff
@@ -586,6 +587,18 @@ const validateForCheckout = (cart) => {
     return {code: 'VALID'};
 };
 
+const mailPendingCarts = async () => {
+    try {
+        const carts = await Cart.find({});
+        for (const cart of carts) {
+            await servicesMail.sendMailPendingCarts(cart);
```

Comment:
The try-catch block need to be in the for-loop. Keep errors in variable and display it at the end